cmd/template/kubernetes.go

```diff
@@ -37,6 +37,15 @@ func buildTemplateKubernetesCore(options *templateCanaryOptions) (*yaml.Node, er
 	root.Content = append(root.Content, util.BuildStringNode("kind", "kubernetes", "")...)
 	root.Content = append(root.Content, util.BuildStringNode("application", "<AppName>", "The name of the application to deploy.")...)
 
+	deployConfigNode, deployConfigValuesNode := util.BuildMapNode("deploymentConfig",
+		"")
```

Comment:
```suggestion
	deployConfigNode, deployConfigValuesNode := util.BuildMapNode("deploymentConfig", "")
```
Minor, seems like this could be on the same line since the comment is empty.